include/bluetooth/direction.h

```diff
@@ -59,4 +140,48 @@ int bt_df_adv_cte_tx_enable(struct bt_le_ext_adv *adv);
  */
 int bt_df_adv_cte_tx_disable(struct bt_le_ext_adv *adv);
 
+/** @brief Enable receive and sampling of Constant Tone Extension for the given
+ *         sync set.
+ *
+ * Receive and sampling of Constant Tone Extension may be enabled only after
+ * periodic advertising sync is established.
+ *
+ * @param[in] sync           Periodic advertising sync object.
+ * @param[in] params         CTE receive and sampling parameters.
+ *
+ * @return Zero on success or (negative) error code otherwise.
+ */
+int bt_df_per_adv_sync_cte_rx_enable(struct bt_le_per_adv_sync *sync,
+				     const struct bt_df_per_adv_sync_cte_rx_param *params);
+
+/** @brief Disable receive and sampling of Constant Tone Extension for the given
+ *         sync set.
+ *
+ * @param[in] sync           Periodic advertising sync object.
+ *
+ * @return Zero on success or (negative) error code otherwise.
+ */
+int bt_df_per_adv_sync_cte_rx_disable(struct bt_le_per_adv_sync *sync);
+
+/**
+ * @brief Register connectionless CTE sampling report callbacks.
+ *
+ * Adds the callback structure to the list of callback structures that monitors
+ * CTE receive and sampling for periodic advertising sync.
+ *
+ * @param cb                 Callback struct. Must point to memory that remains
```

Comment:
Aligning against a different comment block is not needed. I would remove it, it looks better without.

Reply:
I'll correct it.